Dockerfile

```diff
@@ -8,11 +8,7 @@ RUN apt update && apt install -y nginx
 
 COPY nginx.conf /etc/nginx/sites-available/default
 
-COPY package.json ./
-COPY pnpm-lock.yaml ./
-
-FROM base as test
+COPY . .
 
 RUN pnpm install --frozen-lockfile --prefer-offline
 
-COPY . .
```

Comment:
This change will not copy the the files created by `RUN pnpm install --frozen-lockfile --prefer-offline`. Are you sure we don't want them? If that's the case should we delete that line?

Reply:
as far as im aware the pnpm is run once all the files are inside the docker container, since `COPY . .` runs beforehand
so the node_modules are already inside the container